marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -121,6 +121,69 @@ describe('TilesView component', () => {
         expect(document.querySelector('#flows-container')).toBeInTheDocument();
         expect(getByText('Create Flow')).toBeInTheDocument();
         expect(getByText('testFlow')).toBeInTheDocument();
+        // delete should work
+        fireEvent.click(getByTestId('deleteFlow-0'));
```

Comment:
Am I allowed to delete a step as a step-runner. step-runner should not have CRUD access correct?